tests/pretrained_test.py

```diff
@@ -406,8 +406,11 @@ def test_transformer_qa(self):
 
     @pytest.mark.parametrize("model_id, model_card", get_pretrained_models().items())
     def test_pretrained_models(self, model_id, model_card):
-        # Each model in pretrained_models should have an archive and registered_predictor_name.
+        # Each model in pretrained_models should have an archive and a description.
         assert model_card.archive_file is not None
+        assert model_card.display_name is not None
```

Comment:
we should add id to this list

Reply:
A model card cannot be instantiated without `id`. So, no need to check it again.